src/components/steps/general-info/GeneralInfoStep.tsx

```diff
@@ -29,24 +29,34 @@ class GeneralInfoStep extends React.Component<StepProps> {
     render() {
         const { project } = this.props;
         const fields = [
-            getTextField("name", i18n.t("Name"), project.name, {
+            getTextField("name", i18n.t(`Name ${isCompulsory("name")}`), project.name, {
```

Comment:
While conceptually ok, unfortunately it does not work: we cannot use interpolated scripts in `i18n.t`, because the i18n library parses the code to extract i18.t(STRING) the POT file, t won't find this literal. Typical solution: `i18n.t("Name") + (isCompulsory("name") ? " (*)" : "")`